README.md

```diff
@@ -1,41 +1,44 @@
-# risc0-rust-starter
+# RISC Zero Rust Starter
 
-This is a small example designed to be a minimum starting project for the RISC Zero zero knowledge virtual machine.
-In this example, we use zero knowledge proof techniques to prove that we know the factors of some composite number `N`, without revealing what the factors are.
+The `risc0-rust-starter` repo is a minimal starting project for zero-knowledge software development. In this RISC Zero "Hello World" project, we use zero-knowledge proof techniques to prove that we know the factors of some composite number `N`, without revealing what the factors are. 
 
-For the main project, see [here](https://github.com/risc0/risc0)
+For more information, check out the [main risc0 repo](www.github.com/risc0/risc0) and the [RISC Zero website](http://www.RISCZero.com).
 
-# Building & running
+# Quick Start
 
-Make sure you are using the nightly version of rust:
+Make sure you have [Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html) installed and are using the [nightly](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html) version of [Rust](https://doc.rust-lang.org/book/ch01-01-installation.html):
```

Comment:
I'd drop the line about installing Cargo, because it is included and implied with "using a nightly version of rust". Also, there's a nightly build every night, so it's `a` instead of `the`.

Reply:
thanks and fixed